chpl/chpl-resources/src/main/resources/errors.properties

```diff
@@ -360,6 +360,10 @@ surveillance.nonconformity.notFound=Nonconformity not found.
 surveillance.pending.notFound=Pending surveillance not found.
 surveillance.badCharacterFound=An unrecognized character was found for %s. Please check the input data.
 surveillance.newlineCharacterFound=A newline character was found for %s. This will not display on the CHPL website, please consider removing the newline.
+surveillance.inputStream=Could not get input stream for job data string for job with ID %s.
```

Comment:
Theoretically all of these messages should be something a non-technical user could understand. "Input stream" is unlikely to be of much use to them. Even a generic, "something went wrong with Job ID %s, please try again, or contact the CHPL team for support" might be better. Recommend running whatever phrasing you come up with by the Product Owner